drive_cli/dcli.py

```diff
@@ -53,40 +54,41 @@ def go_back(picker):
 def drive_data(*argv):
     try:
         dclipath = os.path.join(dirpath, '.drivecli')
-        if not os.path.isfile(dclipath):
+    except OSError as e:
+        print("Path name error", e)
+    if not os.path.isfile(dclipath):
+        try:
+            with open(dclipath, 'w')as outfile:
```

Comment:
This could have been handled in a better way. Instead of pushing everything inside a block, you could just raise an exception after checking whether the file exist or not.